Misc/NEWS.d/next/Core_and_Builtins/2023-04-29-23-15-38.gh-issue-103997.BS3uVt.rst

```diff
@@ -0,0 +1,4 @@
+String arguments passed to "-c" are now automatically dedented as if by
+:func:`textwrap.dedent`. This allows "python -c" invocations to be indented
+in shell scripts without causing indentation errors. (Patch by Jon Crall and
+Steven Sun)
```

Comment:
```suggestion
in shell scripts without causing indentation errors. Patch by Jon Crall and
Steven Sun.
```

Reply:
?

There's an opening bracket on the line above, it's a matter of preference if you use a full stop for a single sentence in brackets. But removing the closing bracket is definitely wrong.

Reply:
Actually, most of the entries in the changelog are of the form `. Patch by XXX` rather than `. (Patch by...)` (1k+ vs ~40) and I thought that the `(...)` form was only for the `(Contributed by [...])` in the What's New page.

Reply:
> Note: I've removed brackets on both side I think?

Oh yes, I do apologise -- the app was showing this very oddly, I should have waited to check on my computer.

> Actually, most of the entries in the changelog are of the form `. Patch by XXX` rather than `. (Patch by...)` (1k+ vs ~40) and I thought that the `(...)` form was only for the `(Contributed by [...])` in the What's New page.


I don't think it's worth trying to enforce too strict a style for NEWS entries, in reality we're the only people that read them as closly as this. Happy with either style though.

Reply:
Ok, let's not burden this PR with more work. If needed, we'll do a copy-edit.